.github/Hacktoberfest.md

```diff
@@ -0,0 +1,13 @@
+# 🙌 Hacktoberfest
```

Comment:
Per the previous team meeting can we move this into the top of the Contributing.md for the time being !

Reply:
change completed